config/katello.migrations/171026094113-rename-cp-oauth.rb

```diff
@@ -0,0 +1,2 @@
+answers['katello'].delete('oauth_key') if answers['katello'].key?('oauth_key')
```

Comment:
Would it make sense to just use ```if answers['katello']``` since that would also work if the value is false rather than a hash.

Reply:
It should, I lean towards being specific as possible in general.

Reply:
@ekohl Thoughts? if you feel strongly about your edge case, I am fine with that change being made. Otherwise lets ship it.

Reply:
I do think that having more robust code is a good thing and establishes a common pattern but don't feel strongly about this because I also see reasons for this particular case. Let's just merge this as is.